_examples/golang-sse/webapp/src/rpc.gen.ts

```diff
@@ -511,7 +514,9 @@ export interface WebrpcOptions {
 
 export interface WebrpcStreamOptions<T> extends WebrpcOptions {
   onMessage: (message: T) => void;
-  onError: (error: WebrpcError, reconnect?: () => void) => void;
+  onError:
+    | ((error: WebrpcError) => void)
+    | ((error: WebrpcError, reconnect: () => void) => void);
```

Comment:
@kolberg-dev Should we pass `reconnect()` at all times; and throw an error if someone tries to reconnect on abort signal (ie. `WebrpcRequestCanceled` error)?

We could document that `reconnect()` shouldn't be used when the stream is explicitly canceled.

Reply:
https://github.com/webrpc/webrpc/pull/237/commits/89881be53915a71b2157ccc89736eb64265105b5
I dont think this should be a webrpcerror, we should break the app during dev